Client/Application/Migration.swift

```diff
@@ -60,7 +60,13 @@ extension Preferences {
         // Security
         NSKeyedUnarchiver.setClass(AuthenticationKeychainInfo.self, forClassName: "AuthenticationKeychainInfo")
         if let pinLockInfo = KeychainWrapper.standard.object(forKey: "pinLockInfo") as? AuthenticationKeychainInfo {
-            KeychainWrapper.sharedAppContainerKeychain.setAuthenticationInfo(pinLockInfo)
+            //Checks if browserLock was enabled in old app.
+            let isBrowserLockEnabled: Bool = UserDefaults(suiteName: AppInfo.sharedContainerIdentifier)?.bool(forKey: "profile.browserLock") ?? false
```

Comment:
You can use `Preferences.defaultContainer` instead of `UserDefaults(suiteName: AppInfo.sharedContainerIdentifier)`

Reply:
Not huge since I think its always going to be `profile.` for us, but you should use `profile.prefs.getBranchPrefix()` instead of hardcoding `profile.`